cmd/tke-upgrade/README.md

```diff
@@ -0,0 +1,25 @@
+## How To Upgrate TKEStack
+
+### git clone
+```shell script
+git clone http://git.code.oa.com/eetke/upgrade.git
+```
+
+### find installer data in /opt/tke-install/data and copy to tke-upgrade
+```shell script
+cp /opt/tke-installer/data/tke.json cmd/tke-upgrade/app/data/tke.json
+cp /opt/tke-installer/data/oidc_client_secret cmd/tke-upgrade/app/data/oidc_client_secret
```

Comment:
?

Reply:
tkeStack components's upgrade must fetch some cluster infomations where is from tke-installer environment